index.html

```diff
@@ -5,21 +5,110 @@
     	<meta name="keywords" content="one, two, three">
 
-		<title>Blank Template</title>
+		<title>Relaxr | Blog</title>
 
 		<!-- external CSS link -->
-		<link rel="stylesheet" href="css/reset.css">
+		<link rel="stylesheet" href="css/normalize.css">
+
+		<link href="https://fonts.googleapis.com/css?family=Open+Sans:300,300i,400,600,700,800" rel="stylesheet">
+
 		<link rel="stylesheet" href="css/style.css">
 
 	</head>
 	<body>
 		<div id="container">
 			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
+				<div class="clearfix">
+					<div class="header">
+						<h1>Relaxr</h1>
+					</div> <!-- / header -->
+
+					<nav>
+						<a href="about/index.html">About</a>
+
+						<a href="#">FAQ</a>
+
+						<a href="#">Team</a>
+
+						<a href="#">Contact Us</a>
+
+						<a href="index.html">Blog</a>
+					</nav>
+				</div> <!-- / clearfix -->
 			</header>
+
+			<div class="main">
+				<div class="clearfix">
+					<section class="posts">
+						<article>
+							<p>How I Implemented Relaxr in 2 weeks and changed my life.</p>
+
+							<img src="images/blog_photo1.jpg">
+
+							<p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+							tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+							quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+							consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+							cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+							proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+							<p class="readmore">Read More ></p>
+						</article>
+
+						<article>
+							<p>I traveled across the globe thanks to Relaxr’s Excel automation</p>
+
+							<img src="images/blog_photo2.jpg">
+
+							<p class="content">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod
+							tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam,
+							quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo
+							consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse
+							cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non
+							proident, sunt in culpa qui officia deserunt mollit anim id est laborum.</p>
+
+							<p class="readmore">Read More ></p>
+						</article>
+					</section> <!-- posts -->
+
+					<section class="gutter">
+
+					</section> <!-- gutter -->
+
+					<section class="sidebar">
+						<p>Categories</p>
+
+						<ul class="list">
+							<li>Success stories</li>
+							<li>Stats</li>
+							<li>How tos</li>
+							<li>Best of</li>
+							<li>Business tips</li>
+						</ul>
+
+						<p>About Relaxr</p>
+
+						<p class="abouttext">Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla. <br><span>Learn More</span></br></p>
```

Comment:
Two things: Break tags `<br>` do not have a closing tag and I would not have used a <br> tag here, I would have used CSS to modify and add extra space.